src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -66,7 +67,8 @@ TopicDomainHeader.propTypes = {
   hideSeeMember: PropTypes.bool,
   isFollow: PropTypes.bool,
   memberCount: PropTypes.number,
-  isLoading: PropTypes.bool
+  isLoading: PropTypes.bool,
+  followType: PropTypes.string
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `followType` has been added to PropTypes validation but it's not clear what values this prop can take. It would be better to use `PropTypes.oneOf([...])` to specify the possible string values that `followType` can have.
```diff
- followType: PropTypes.string
+ followType: PropTypes.oneOf(['signed', 'incognito', 'default'])
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->